algosdk/atomic_transaction_composer.py

```diff
@@ -174,16 +379,14 @@ def add_method_call(
         method: abi.Method,
         sender: str,
         sp: transaction.SuggestedParams,
-        signer: "TransactionSigner",
-        method_args: Optional[
-            List[Union[Any, "TransactionWithSigner"]]
-        ] = None,
+        signer: TransactionSigner,
+        method_args: Optional[List[Union[Any, TransactionWithSigner]]] = None,
         on_complete: transaction.OnComplete = transaction.OnComplete.NoOpOC,
         local_schema: Optional[transaction.StateSchema] = None,
         global_schema: Optional[transaction.StateSchema] = None,
         approval_program: Optional[bytes] = None,
         clear_program: Optional[bytes] = None,
-        extra_pages: Optional[int] = None,
+        extra_pages: int = 0,
```

Comment:
downstream, the optionality of the`extra_pages` params is signalted with `0` so I'm unifying the API here.